core/templates/components/skill-selector/select-skill-modal.component.ts

```diff
@@ -69,5 +73,17 @@ export class SelectSkillModalComponent extends ConfirmOrCancelModal {
 
   setSelectedSkillId(skillId: string): void {
     this.selectedSkillId = skillId;
+    this.isSaveButtonEnabled();
```

Comment:
Why is this method call made here? It looks like the return value is not used.

Reply:
You were right calling this method was unnecessary, made some new commits.

Thanks